include/uapi/linux/tc_act/tc_skbedit.h

```diff
@@ -30,9 +30,11 @@
 #define SKBEDIT_F_MARK			0x4
 #define SKBEDIT_F_PTYPE			0x8
 #define SKBEDIT_F_MASK			0x10
+#define SKBEDIT_F_INHERITDSFIELD	0x20
 
 struct tc_skbedit {
 	tc_gen;
+	__u32 flags;
```

Comment:
Use `__u64` instead of `__u32` like in `struct tc_skbmod`. The motivation for `__u64` is that if more than 32 flags a needed, there's no need to change the interface between the kernel and user space.

Reply:
I use ``__u32`` since in the source code of ``skbedit``, it uses ``__u32`` for the ``flags`` parameter. For example, ``tcf_skbedit_init()`` defines ``u32 flags = 0``; in ``struct tcf_skbedit`` it also uses ``u32 flags``. If we change it to ``__u64``, we may change several other places. What do you think?

Reply:
I would only change in `struct tc_skbedit` because this is part of the interface between kernel and user space. Consider when someone has to add the 33th flag, only the kernel code will need to change. In summary, I'm trying to keep the kernel-user interface as stable as possible while allowing us to change whatever we need in the kernel side.

Reply:
I see. Done.